orchagent/port.h

```diff
@@ -18,12 +18,15 @@ extern "C" {
 #include <sairedis.h>
 
 #define DEFAULT_PORT_VLAN_ID    1
+
+#define MAX_MACSEC_SECTAG_SIZE 32
+
 /*
  * Default MTU is derived from SAI_PORT_ATTR_MTU (1514)
  * Orchagent adds extra 22 bytes for Ethernet header and FCS,
  * hence setting to 1492 (1514 - 22)
  */
-#define DEFAULT_MTU             1492
+#define DEFAULT_MTU             (1492 - MAX_MACSEC_SECTAG_SIZE)
```

Comment:
I dont think we should be changing this default value as there could be multple dependencies/test making this assumption.

Reply:
I see it is used to set default m_mtu value. Later the code in getPortMtu reads it on start from SAI and substracts MAX_MACSEC_SECTAG_SIZE. 
The default for SAI_PORT_ATTR_MTU is defined by headers, so it always sets m_mtu to the value 1492 - MAX_MACSEC_SECTAG_SIZE, thus theoretically we can save SAI call per port if we set it here to a constant.

Not sure if that was the intention to substract macsec tag size regardless of whether macsec is enabled or not but it is current behaviour.

Reply:
@prgeor It is related. We have two flows of port initialization - 
1. SAI starts with default ports and we get MTU in getPortMtu() which calls SAI get API.
2. We create ports not providing MTU value. In this case the MTU should be default as defined by SAI headers. So I avoid getting it from SAI.
This define is used to initialize m_mtu variable. Since getPortMtu() always substracts MAX_MACSEC_SECTAG_SIZE I updated this define to match the value that would have been returned by getPortMtu().

Reply:
Discussed online with @prgeor : keep the mtu change seperate from this PR.
Initial MTU may be diifferent from SAI spec on some platforms.